components/sup/src/manager/mod.rs

```diff
@@ -35,12 +38,14 @@ use toml;
 
 pub use manager::service::{Service, ServiceConfig, ServiceSpec, UpdateStrategy, Topology};
 use self::service_updater::ServiceUpdater;
-use error::Result;
+use error::{Error, Result};
 use config::GossipListenAddr;
 use manager::census::{CensusUpdate, CensusList, CensusEntry};
 use manager::signals::SignalEvent;
 use http_gateway;
 
+pub const DATA_PATH: &'static str = "/hab/sup/data";
```

Comment:
I think I'll be tweaking this right quick but totally fine for the moment. I'll still make sure that the directory exists before the manager starts so it can load its member data, etc.